content/en/users/check-in/vos/_index.md

```diff
@@ -9,6 +9,23 @@ description: >
 This page contains information about using Check-in for managing your Virtual
 Organisation (VO).
 
+## Background
+
+In simple terms a Virtual Organisation (VO) is just a group of users. In
```

Comment:
Here, or before in the page if it's not done anywhere, you could link to the VO definition in the glossary: https://ims.egi.eu/display/EGIG/Virtual+organisation

Reply:
Good idea!